Admin/Extension/RouteReferrersExtension.php

```diff
@@ -1,44 +0,0 @@
-<?php
-
-/*
- * This file is part of the Symfony CMF package.
- *
- * (c) 2011-2015 Symfony CMF
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-namespace Symfony\Cmf\Bundle\RoutingBundle\Admin\Extension;
-
-use Sonata\AdminBundle\Admin\AdminExtension;
-use Sonata\AdminBundle\Form\FormMapper;
-use Sonata\CoreBundle\Form\Type\CollectionType;
-
-/**
- * Admin extension to add routes tab to content implementing the
- * RouteReferrersInterface.
- *
- * @author David Buchmann <mail@davidbu.ch>
- */
-class RouteReferrersExtension extends AdminExtension
-{
-    public function configureFormFields(FormMapper $formMapper)
-    {
-        $formMapper
-            ->with('form.group_routes', array(
-                'translation_domain' => 'CmfRoutingBundle',
-            ))
-            ->add(
-                'routes',
-                CollectionType::class,
-                array(),
-                array(
-                    'edit' => 'inline',
-                    'inline' => 'table',
-                )
```

Comment:
Got a [hint](http://iltar.nl/) but it will be a more complex work. I really like his idea, used that in a projekt last year. The DTO should stay with the form types inside ther routing-bundle,right?